task1/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "bucket_name" {
+    type        = string
+    description = "Value of the Name tag for the S3 Bucket"
+    default     = "First TF Bucket"
+}
+
+variable "project_name" {
+    type        = string
+    description = "Value of the Project tag for the S3 Bucket"
+    default     = "TF Setup"
+}
+
+variable "proj_environment" {
+    type        = string
+    description = "Value of the Project tag for the S3 Bucket"
+    default     = "Dev"
+}
+
+variable "availability_zone_names" {
+    type    = list(string)
+    default = ["us-east-1"]
+    description = "The AZ to be used"
```

Comment:
us-east-1 is a region name, not az.
usually we only setup a default region for the provider. and don't make a list